schemas/telemetry/account-ecosystem/account-ecosystem.4.schema.json

```diff
@@ -1,5 +1,15 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
+  "mozPipelineMetadata": {
+    "bq_dataset": "account_ecosystem",
+    "bq_table": "desktop",
```

Comment:
Is the intent here to map telemetry account ecosystem pings to a non-telemetry dataset/table? Certainly that is a capability we're trying to support via custom metadata, but in this case we would probably have MSG write the .bq versions of these to 

`schemas/account-ecosystem/desktop/desktop.4.bq`

in `generated-schemas`, and downstream logic would remain the same. (It's my understanding that aet pings aren't sensitive once the identifier is decrypted so this is perhaps just for demonstration purposes).

Reply:
> Is the intent here to map telemetry account ecosystem pings to a non-telemetry dataset/table? 

Yes, that was what I was intending to demonstrate here.

> Certainly that is a capability we're trying to support via custom metadata, but in this case we would probably have MSG write the .bq versions of these to
> 
> `schemas/account-ecosystem/desktop/desktop.4.bq`
> 
> in `generated-schemas`, and downstream logic would remain the same. (It's my understanding that aet pings aren't sensitive once the identifier is decrypted so this is perhaps just for demonstration purposes).

Yes, it's my intention that MSG would read this metadata and write out the BQ schema to a path determined by these fields.